website/docs/introduction/whats-marp.md

```diff
@@ -43,44 +43,42 @@ bar
 
 [markdown]: https://en.wikipedia.org/wiki/Markdown
 [commonmark]: https://commonmark.org/
 
-By cherry-picking additional features, we are make careful to keep compatibility with the general Markdown document. It means the result of rendering keeps looking good even if you editting and previewing the Marp Markdown in a general Markdown editor.
-
 ### Theme CSS
 
-Marp ecosystem is faithful to Web basics, HTML and CSS. We provide a theming system using plain CSS to style your slides. It is mostly same as styling HTML page so you can jump right in by just learning CSS.
+The Marp ecosystem is designed to be intuitive to anyone who has made a webpage. As long as you know HTML and CSS, you should be able to style your presentation easily. Our theming system allows you to use plain CSS to style your slides.
 
-"[Separation of content and style](https://en.wikipedia.org/wiki/Separation_of_content_and_presentation)" is an important thing also in the slide deck. You can apply the design made by us or community into your deck easily. If necessary, you can tweak styles by inline style within Markdown too.
+Marp is designed with the "[Separation of content and style](https://en.wikipedia.org/wiki/Separation_of_content_and_presentation)" in mind. The goal is to make it easy for users to apply designs made by the community to the user's content.
 
 ### Export to PDF, PPTX, HTML
 
-We have a first-class support of conversion into other file formats. Marp has given importance to reproducibility of rendering so you do not need to worry about broken layout.
-
-Marp is not aim to giving a presentation with itself. Exporting to PDF format is the best choice to give a presentation. You can present in anywhere, any devices, and regardless network status by just bringing a one PDF file.
+Marp has first-class support for conversion into other file formats. We prioritize reproducible rendering across formats so that users do not have to worry about different formats breaking layouts. Our goal is to make sure the PDF, PPTX, and HTML versions of your slides look exactly the same.
 
-If you want more familiar way to make a presentation, you can export to PowerPoint document (PPTX) too. For using interactive contents or serving the deck on the web, you also can export to HTML that have bare minimum features for presentation.
+Note that Marp is not designed to be stand-alone presentation software. The simplest way to present is to export to HTML and open the presentation in a browser. The HTML format allows you to serve your slide deck on the internet. Also, it allows you to show interactive content and make use of features like fragmented lists. If you are presenting without access to the internet, you should export to the PDF format. If you want to add additional content manually, Marp also allows you to export to the PowerPoint (PPTX) format. 
```

Comment:
> The simplest way to present is to export to HTML and open the presentation in a browser.

Why changed an original meaning? Marp team is thinking PDF format is the best choice rather than HTML. Surely HTML export and its interactivity and still important for competition with other presentation tools, but we have believed portability by a single PDF file is valuable since made [an original Marp app](https://github.com/yhatt/marp).

> Exporting to PDF format is the best choice to give a presentation. You can present in anywhere, any devices, and regardless network status by just bringing a one PDF file.

Reply:
@yhatt I've switched the recommendation back.

I originally made the change because I thought the HTML format was the preferred format. I thought that this was "HTML-first" because the HTML format seems to be the most "feature-complete" export format. If I'm not mistaken, fragmented lists and speaker notes only work in HTML.

(When I have more time, I will write out an issue about getting speaker notes into PDFs. See https://github.com/pdfpc/pdfpc, which allows speaker notes to be stored as a JSON in a separate file.)